JavaTargetPractise.java

```diff
@@ -107,16 +112,16 @@ public void mouseClicked(MouseEvent e)
             }
         }
 
+        //System.out.println(labelNumber);
         try
         {
-           Object selectLabel = selectedPanel.getComponent(index);
-           JLabel hitLabel = (JLabel) selectLabel;
```

Comment:
used the getComponents() to get an array of components present in the selected panel.
Used explicit casting to get a JLabel object so that i could change the text on the JLabel
I could have used the graphics class but the question required me not to